vortexasdk/endpoints/vessel_movements.py

```diff
@@ -42,6 +42,7 @@ def search(
         filter_products: Union[str, List[str]] = None,
         filter_vessels: Union[str, List[str]] = None,
         filter_vessel_classes: Union[str, List[str]] = None,
+        filter_vessel_status: str = "vessel_status_ballast",
```

Comment:
I think the default argument here should be None rather than defaulting to "vessel_status_ballast".

We can also annotate the type here.

Reply:
Yes yes of course should be `None`. Will correct

Reply:
Done (I am not sure though about the annotation you mentioned).